modules/ozoneBidAdapter.js

```diff
@@ -652,24 +921,39 @@ export const spec = {
     }
     return null;
   },
+  /**
+   * Locate the bid inside the arrBids for this bidId, then discover the video context, and return it.
+   * IF the bid cannot be found return null, else return a string.
+   * @param bidId
+   * @param arrBids
+   * @return string|null
+   */
   getVideoContextForBidId(bidId, arrBids) {
     let requestBid = this.getBidRequestForBidId(bidId, arrBids);
     if (requestBid != null) {
       return deepAccess(requestBid, 'mediaTypes.video.context', 'unknown')
     }
     return null;
   },
+  /**
+   * This is used for cookie sync, not auction call
+   *  Look for pubcid & all the other IDs according to http://prebid.org/dev-docs/modules/userId.html
+   *  @return map
+   */
   findAllUserIds(bidRequest) {
     var ret = {};
-    let searchKeysSingle = ['pubcid', 'tdid', 'idl_env', 'criteoId', 'lotamePanoramaId', 'fabrickId'];
+    // @todo - what is Neustar fabrick called & where to look for it? If it's a simple value then it will automatically be ok
```

Comment:
it is fabrickid.

Also please prefer the eids object, a future release might delete the userids object

Reply:
You're welcome to continue to use userid, we'll make sure everything is in eids before deleting support for other methods in perhaps a year or two. We're just trying to reduce redundancy over time

Reply:
Are you sure it's not there? That's a bug 

https://github.com/prebid/Prebid.js/blob/e3ea28063e5228370faaabe6497515be11b8ce4f/modules/pairIdSystem.js#L86